### DISCUSSION TITLE
Checkbox item border radius

### DISCUSSION THREAD

**Discussion by LankieB** [heroui-inc/heroui#702] [Help] [answered]
Please how do you override the border radius of the checkbox item? I have tried so many options but none seems to be working

**Comment by jrgarciadev** [comment, marked as answer]
Hey @LankieB here's an example: https://codesandbox.io/s/sandpack-project-forked-jznnov?file=/App.js

I also submitted a PR to make the checkbox container customization a bit easier

https://github.com/nextui-org/nextui/pull/709

**Reply by LankieB** [reply]
Thanks so much, Junior for this. This made my day 🤗
